python_src/src/lamp_py/performance_manager/gtfs_utils.py

```diff
@@ -209,46 +211,30 @@ def add_parent_station_column(
     return events_dataframe
 
 
-def remove_bus_records(
-    events_dataframe: pandas.DataFrame,
-    db_manager: DatabaseManager,
-) -> pandas.DataFrame:
+def rail_routes_from_timestamp(
+    timestamp: float, db_manager: DatabaseManager
+) -> List[str]:
```

Comment:
nitpick: since the `timestamp` value here is derived from the S3 paths, it would be a little cleaner to just pass the `str` or `List[str]` of paths into this function and get the rout_id's back. 

Then this function could be called from directly in `get_tu_dateframe_chunks` and `get_vp_dateaframe` and we don't have to pass a `rout_ids` list all around